app/src/main/java/com/android/shelfLife/model/newhousehold/HouseHoldRepository.kt

```diff
@@ -46,9 +49,8 @@ interface HouseHoldRepository {
    * Initializes households by fetching them from Firestore and updating the local cache.
    *
    * @param householdIds List of household IDs to fetch.
-   * @param selectedHouseholdId The ID of the selected household.
    */
-  suspend fun initializeHouseholds(householdIds: List<String>, selectedHouseholdUid: String)
+  suspend fun initializeHouseholds(householdIds: List<String>, selectedHouseholdUid: String?)
```

Comment:
as there is not necessarily a selected household, the type the selectedHouseholdUid need to be String?.